app/src/main/java/com/example/lemonade/ui/theme/Theme.kt

```diff
@@ -126,3 +126,23 @@ fun AppTheme(
         content = content
     )
 }
+
+/**
+ * Sets up edge-to-edge for the window of this [view]. The system icon colors are set to either
+ * light or dark depending on whether the [darkTheme] is enabled or not.
+ */
+private fun setUpEdgeToEdge(view: View, darkTheme: Boolean) {
+    val window = (view.context as Activity).window
+    WindowCompat.setDecorFitsSystemWindows(window, false)
+    window.statusBarColor = Color.Transparent.toArgb()
+    val navigationBarColor = when {
+//        Build.VERSION.SDK_INT >= 29 -> Color.Transparent.toArgb()
+        Build.VERSION.SDK_INT >= 26 -> Color(0xFF, 0xFF, 0xFF, 0x63).toArgb()
```

Comment:
Could we have translucent white or black bottom navigation/ system bar depending on the app theme or this is SDK specific?

Reply:
You can do so if you set those colors, For the status bar, it's API 23+. For the nav bar, it's API 26+.

Reply:
Thank you